collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/audit/TrackChangesReasonTest.kt

```diff
@@ -161,4 +161,15 @@ class TrackChangesReasonTest {
             .clickSave(FormEntryPage("Track Changes Reason"))
             .assertQuestion("What up?")
     }
+
+    @Test
+    fun fillingABlankForm_andClickingSave_andClickingSaveAndExit_doesNotPromptForReason() {
```

Comment:
I'd managed to forget about this case when refactoring and introduced a bug. This test should prevent that happening again!